### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by hjung4** [comment]
@googlebot I signed it!

**Review by ymodak — DISMISSED** [review]

**Review by mihaimaruseac — DISMISSED** [review]

**Review by jpienaar — DISMISSED** [review]

File: tensorflow/core/kernels/depthwise_conv_grad_op.cc [hunk 1]
```diff
@@ -305,7 +305,7 @@ static void ComputeBackpropInput(const DepthwiseArgs& args,
 
   for (int i = 0; i < output_vectorized_size; i += kPacketSize) {
     // Reset accumulator.
-    auto vaccum = Eigen::internal::pset1<Packet>(static_cast<T>(0));
+    auto vacuum = Eigen::internal::pset1<Packet>(static_cast<T>(0));
```
Comment (rthadur) [inline tensorflow/core/kernels/depthwise_conv_grad_op.cc]: Here is the internal review comments 

`No. vaccum is short for "vector accumulator".` can you please rename this

File: tensorflow/core/kernels/depthwise_conv_op.cc [hunk 2]
```diff
@@ -99,7 +99,7 @@ struct DepthwiseConv2DKernel {
 
     for (int i = 0; i < output_vectorized_size; i += kPacketSize) {
       // Reset accumulator.
-      auto vaccum = Eigen::internal::pset1<Packet>(static_cast<T>(0));
+      auto vacuum = Eigen::internal::pset1<Packet>(static_cast<T>(0));
```
Comment (rthadur) [inline tensorflow/core/kernels/depthwise_conv_op.cc]: same as above

File: tensorflow/core/kernels/eigen_pooling.h [hunk 3]
```diff
@@ -352,13 +352,13 @@ struct AvgPoolMeanReducer {
 
   template <typename Packet>
   EIGEN_DEVICE_FUNC EIGEN_STRONG_INLINE Packet
-  finalizePacket(const Packet& vaccum) const {
-    return pdiv(vaccum, packetCount_);
+  finalizePacket(const Packet& vacuum) const {
```
Comment (rthadur) [inline tensorflow/core/kernels/eigen_pooling.h]: same as above

**Review by rthadur — DISMISSED** [review]

**Comment by hjung4** [comment]
Changes have been made.

**Review by mihaimaruseac — APPROVED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**